apps/papra-server/src/modules/documents/documents.routes.ts

```diff
@@ -422,3 +425,53 @@ function setupDeleteAllTrashDocumentsRoute({ app, config, db }: RouteDefinitionC
     },
   );
 }
+
+function setupUpdateDocumentContentRoute({ app, db }: RouteDefinitionContext) {
+  app.put(
+    '/api/organizations/:organizationId/documents/:documentId/content',
+    validateParams(z.object({
+      organizationId: z.string().regex(organizationIdRegex),
+      documentId: z.string(),
+    })),
```

Comment:
Patch instead of put, as we don't override the document 

```suggestion
  app.patch(
    '/api/organizations/:organizationId/documents/:documentId',
    validateParams(z.object({
      organizationId: z.string().regex(organizationIdRegex),
      documentId: z.string(),
    })),
```